randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1603,78 +1411,34 @@ Extra - asset_id: collision_camera_018
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Dock to Big Hub
-      Morph Ball and After Dairon - Central Unit
+      Any of the following:
+          Flash Shift or Flash Shift Skip (Beginner) or Lay Cross Bomb or Use Spin Boost
+          Morph Ball and After Dairon - Central Unit
```

Comment:
It can probably just be Flash or Flash Skip, unless we want cross and spin to be trivial rather than flash skips.